charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: 1.6.0
+appVersion: 1.5.0
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 2.6.0
+version: 2.5.1
```

Comment:
You need to increase the helm chart version to 2.6.1.  Decreasing it will still have the latest version be broken.

Reply:
Working on it